src/ResultsHelpers.ts

```diff
@@ -0,0 +1,839 @@
+    return false;
+  } else if (result === null || result === undefined) {
+    // Specifically no result
+    return false;
+  }
+  return true;
+}
+
+/**
+ * Iterate over episode results, call _buildPopulationRelevanceMap for each result then OR population relevances
+ * together so that a patient level population relevance set can be made.
+ *
+ * @private
+ * @param {EpisodeResults[]} episodeResultsSet - Results for each episode
+ * @returns {PopulationResult[]} List denoting if population calculation was considered/relevant in any episode
+ */
+export function buildPopulationRelevanceForAllEpisodes(
+  populationGroup: R4.IMeasure_Group,
+  episodeResultsSet: EpisodeResults[]
+): PopulationResult[] {
+  const masterRelevanceResults: PopulationResult[] =
+    populationGroup.population?.map(population => {
+      return <PopulationResult>{
+        populationType: MeasureHelpers.codeableConceptToPopulationType(population.code),
+        result: false
+      };
+    }) || []; // Should not end up becoming an empty list.
+
+  episodeResultsSet.forEach(episodeResults => {
+    const episodeRelevance = buildPopulationRelevanceMap(episodeResults.populationResults);
+    masterRelevanceResults.forEach(masterPopResults => {
+      // find relevance in episode and if true, make master relevance true, only if not already true
+      if (masterPopResults.result === false) {
+        if (getResult(masterPopResults.populationType, episodeRelevance) === true) {
+          masterPopResults.result = true;
+        }
+      }
+    });
+  });
+  return masterRelevanceResults;
+}
+
+/**
+ * Builds the relevance results for each population. This creates the `populationRelevance` list that is put on
+ * DetailedPopulationGroupResult
+ *
+ * For each population in the group it is set to true if the population was relevant/considered, false if
+ * NOT relevant/considered. This is used later to determine which define statements are relevant in the calculation.
+ *
+ * For example: If they aren't in the IPP then they are not going to be considered for any other population and all other
+ * populations will be marked NOT relevant.
+ *
+ * This function is extremely verbose because this is an important and confusing calculation to make. The verbosity
+ * was kept to make it more maintainable.
+ *
+ * @param {PopulationResult[]} result - The population results list for the population results.
+ * @returns {PopulationResult[]} The population relevance set.
+ */
+export function buildPopulationRelevanceMap(results: PopulationResult[]): PopulationResult[] {
+  // Create initial results starting with all true to create the basis for relevance.
+  const relevantResults: PopulationResult[] = results.map(result => {
+    return {
+      populationType: result.populationType,
+      result: true
+    };
+  });
+
+  /** TODO: Determine how Straifiers play here
+  // If STRAT is 0 then everything else is not calculated
```

Comment:
Again asking about the TODO: is this something we need to address?

Reply:
Stratifiers are handled at a different level now. This has been removed in 2284c38.